pkgs/by-name/ob/objfw/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  autoconf,
+  autogen,
+  automake,
+  clang,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "objfw";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "ObjFW";
+    repo = "ObjFW";
+    rev = "refs/tags/1.1.7-release";
+    hash = "sha256-0ylG/2ZSO3b8zdh6W9QJH5OJW9V344CCik1DduV5mhI=";
+  };
+
+  nativeBuildInputs = [
+    clang
```

Comment:
Missed this, to use clang you should do `clangStdenv.mkDerivation`

Reply:
Ah, thanks, I didn't realise that. Should I open a new PR for this? or should I do that in a future PR when I version bump?

Reply:
Open a new pr